src/homework6/Chamomile.java

```diff
@@ -0,0 +1,14 @@
+package homework6;
+
+public class Chamomile extends Flower {
+
+
+    @Override
+    public String toString() {
+        return "Chamomile";
+    }
+
+    public Chamomile(int price) {
```

Comment:
constructors may be before another methods
[Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)